source/server/admin/stats_render.cc

```diff
@@ -53,8 +53,8 @@ void StatsTextRender::addDetail(const std::vector<Stats::ParentHistogram::Bucket
                                 Buffer::Instance& response) {
   absl::string_view delim = "";
   for (const Stats::ParentHistogram::Bucket& bucket : buckets) {
-    response.addFragments(
-        {delim, absl::StrCat(bucket.lower_bound_, ",", bucket.width_, ":", bucket.count_)});
+    response.addFragments({delim, absl::StrFormat("%.15g,%.15g:%lu", bucket.lower_bound_,
```

Comment:
Note: I did not use the new mechanism to serialize a double in this case because it did not make the benchmark run faster, but did make the code more verbose and harder to read. However I realized we were dropping precision on the text histogram so I wanted to fix it.